libafl_qemu/build_linux.rs

```diff
@@ -232,4 +234,31 @@ pub fn build() {
                 .success()
         );
     }
+
+    if cfg!(feature = "usermode") && rasan {
+        let rasan_dir = Path::new("librasan");
+        let rasan_dir = fs::canonicalize(rasan_dir).unwrap();
+        println!("cargo:rerun-if-changed={}", rasan_dir.display());
+
+        let mut just = Command::new("just");
+        just.env("ARCH", cpu_target);
+        just.env("TARGET_DIR", &target_dir);
+        if cfg!(debug_assertions) {
+            just.env("PROFILE", "dev");
+        } else {
+            just.env("PROFILE", "release");
+        }
+        assert!(just
```

Comment:
Maybe add some informative text to the assert

Reply:
Done.